.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
+- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
+- [ ] Gallery example in `./doc/examples` (new features only)
+- [ ] Unit tests
+
```

Comment:
Or we can keep the short checklist and enrich the guidelines with more specific details.